src/react-app-env.d.ts

```diff
@@ -1 +1,11 @@
 /// <reference types="react-scripts" />
+declare module "*.jpg";
+declare module "*.png";
+declare module "*.jpeg";
+declare module "*.gif";
+declare module "*.mp4";
+declare module "*.mp3";
+declare module "*.webm";
+declare module "*.pdf";
+declare module "*.gltf";
```

Comment:
What’s this for?

Reply:
so typescript is able to see that it's a valid data type for example .gltf is a 3d file that uses a loader, but ts will flag it as an error because it will not understand that its a valid type.